### ISSUE TITLE
V4.0.1 breaks with error 'Cannot read property 'install' of undefined'

### ISSUE THREAD

**Issue by DaveAuld** [probil/vue-socket.io-extended#430]
This standard code fails to build with error thrown on the Vue.use line
![Capture](https://user-images.githubusercontent.com/4674288/70708463-9f13f900-1ceb-11ea-8e99-3a6b1f7be17b.PNG)
, however forcing down to **V4.0.0** works fine. This is on CodeSandbox using Vue 2.6.10 and  socket.io-client v2.3 
```
import Vue from "vue";
import App from "./App.vue";
import VueSocketIOExt from "vue-socket.io-extended";
import io from "socket.io-client";

//Central store for all local data
import store from "./ChatStore";

//Initialise socket
const socket = io("https://xxxxxxxxxxx/");

//Bind to Vue
Vue.use(VueSocketIOExt, socket);

Vue.config.productionTip = false;

new Vue({
  store,
  render: h => h(App)
}).$mount("#app");
```

**Comment by probil** [comment]
Thanks, @DaveAuld 
I'll take a look ASAP

**Comment by master83** [comment]
any luck or update on this? Thanks!

**Comment by probil** [comment]
It has been fixed in v4.1, v4.2 work fine. 

Proof:
[![Edit Vue Socket IO Extended Bug #430 reproduction](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/vue-socket-io-extended-bug-430-reproduction-ryiql?fontsize=14&hidenavigation=1&theme=dark)

Closing the issue...
